src/app/components/account-details/account-details.component.html

```diff
@@ -385,7 +385,7 @@ <h3 class="uk-margin-remove-bottom">Transactions</h3>
             </ng-template>
           </ng-template>
           <td class="amount-column uk-text-middle uk-text-muted" [title]="('Incoming Transaction') + ( (pending.amountRaw && (pending.amountRaw > 0) ) ? ( ', +' + ( pending.amountRaw.toString(10) ) + ' raw' ) : '' )">
-            <span class="uk-text-small">Ready to receive</span><br>
+            <span class="uk-text-small">{{ 'account-details.ready-to-receive' | transloco }}</span><br>
```

Comment:
could use `'general.actions.ready-to-receive'` here

Reply:
Fixed, thanks!